analysis/evaluation_analysis/eval_analysis.py

```diff
@@ -0,0 +1,277 @@
+import streamlit as st
```

Comment:
can you rename the dataset files to be separated with `_`?

instead of `Human Eval Dataset.csv` -> `Human_Eval_Dataset.csv`